src/scripts/basicContext.js

```diff
@@ -1,264 +1,239 @@
-let overflow = null
+let overflow = null;
```

Comment:
I assume, the only changes here are due to formatting, i.e. changed indention, white spaces, missing semi-colons at line endings, etc. pp. Right?

But again, we should use tabulators not spaces for indention.